logistics/scripts/modules/TestHarness.psm1

```diff
@@ -61,6 +61,18 @@ function Get-TestHarnessExecutable {
     return $testHarnessExecutable
 }
 
+function Invoke-SdkGenConsole {
+    Param(
+        [string] $buildConfiguration = "Debug"
+    )
+    
+    $sdkGenConsoleExecutableFolder = "$(Get-RepositoryResolvedPath "\Utilities\SdkGen\EdFi.SdkGen.Console")\bin\$buildConfiguration\netcoreapp3.1"
```

Comment:
not sure if we should hard code the full location. why not search for the file using a glob pattern?